src/server/server.js

```diff
@@ -28,6 +28,10 @@ var express = require('express'),
     cookieParser = require('cookie-parser'),
     indexTpl = dot.template(fs.readFileSync(path.join(__dirname, '..', 'client', 'netsblox.dot')));
 
+if (proces.env.ENV === 'production') {
```

Comment:
This should be `process`. It might be cleaner to set this when defining `DEFAULT_OPTIONS` initially (rather than setting `vantage` to `true`